.github/workflows/h5py.yml

```diff
@@ -21,7 +21,7 @@ jobs:
 
     - name: Run a multi-line script
       run: |
-        sed -i 's/hdf5@1.10.4:1.14/hdf5@1.10.4:/g' \
+        sed -i 's/hdf5@1.10.6:1.14/hdf5@1.10.6:/g' \
```

Comment:
What exactly does this step do?

Reply:
I think this extends the required range for hdf5 from 1.10.6 - 1.10.14 to any version 1.10.6 or greater.